package.json

```diff
@@ -1,60 +1,90 @@
 {
   "name": "sanity-plugin-fieldwork",
-  "version": "1.1.2",
+  "version": "1.0.0",
   "description": "Inspect your Sanity.io schema fields straight in your Studio using this tool",
-  "main": "lib/index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/ahmdigital/fieldwork.git"
-  },
-  "scripts": {
-    "build": "babel src -d lib --copy-files",
-    "lint": "eslint src",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
   "keywords": [
-    "cms",
-    "content-model",
-    "content",
-    "headless",
-    "modelling",
-    "realtime",
-    "sanity-io",
-    "sanity-tool",
     "sanity",
-    "tool",
-    "schema",
-    "fields"
+    "sanity-plugin"
   ],
-  "author": "ahm digital (Cam Chamberlain)",
-  "license": "MIT",
+  "homepage": "https://github.com/ahmdigital/fieldwork#readme",
   "bugs": {
     "url": "https://github.com/ahmdigital/fieldwork/issues"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/ahmdigital/fieldwork.git"
+  },
+  "license": "MIT",
+  "author": "ahm digital <joshua.edwards@medibank.com.au>",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "source": "./src/index.ts",
+      "import": "./dist/index.esm.js",
+      "require": "./dist/index.js",
+      "default": "./dist/index.esm.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.esm.js",
+  "source": "./src/index.ts",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist",
+    "sanity.json",
+    "src",
+    "v2-incompatible.js"
+  ],
+  "scripts": {
+    "build": "run-s clean && plugin-kit verify-package --silent && pkg-utils build --strict && pkg-utils --strict",
+    "clean": "rimraf dist",
+    "format": "prettier --write --cache --ignore-unknown .",
+    "link-watch": "plugin-kit link-watch",
+    "lint": "eslint .",
+    "prepublishOnly": "run-s build",
+    "test": "jest",
+    "watch": "pkg-utils watch --strict"
+  },
   "dependencies": {
+    "@sanity/incompatible-plugin": "^1.0.4",
     "lodash": "^4.17.19",
-    "prop-types": "^15.7.2",
-    "react": "^16.13.1",
-    "react-icons": "^3.10.0",
-    "styled-components": "^5.1.0"
-  },
-  "peerDependencies": {
-    "@sanity/base": "^2.0.0"
+    "react-icons": "5.0.1"
```

Comment:
regular deps in a library/plugin should have `^`s so the installer can BYO any v5 version